documentation/api/query/v4/environments.markdown

```diff
@@ -35,6 +41,25 @@ See [the Operators page](./operators.html)
 
 * `"name"` (string): the name of an environment
 
+### Subquery Relationships
+
+Here is a list of relationships that can be queried using implicit subqueries. For
+more information consult the documentation for [subqueries].
+
+Direct relationships:
+
+* [factsets][`factsets`]: Factsets received from this environment.
+* [reports][`reports`]: Reports received from this environment.
+* [catalogs][`catalogs`]: Catalogs received for this this environment.
+
+Transitive relationships:
+
+* [facts][`facts`]: Fact names and values received from this environment.
+* [fact-contents][`fact_contents`]: Factset paths and values received from this environment.
+* [events][`events`]: Report events triggered from this environment.
+* [resources][`resources`]: Catalog resources received for this ebvuribnebt.
```

Comment:
Hahaha ... I can only presume I lost my home key positioning, and was typing without looking at the screen :-). Thats fabulous.